src/Components/Calculator.js

```diff
@@ -1,43 +1,58 @@
-import React from 'react';
-import CalculatorSign from './Calulatorsign';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+
+function AppCalculator() {
+  const [state, setState] = useState({ total: 0, next: null, operation: null });
+
+  const clickHandler = (e) => {
+    setState((state) => calculate(state, e.target.name));
+  };
+
+  return (
+    <div className="maincalculator">
+
+      <div className="showscreen">
+        {state.total}
+        {state.operation}
+        {state.next}
+      </div>
+
+      <div className="symbole">
+        <button onClick={clickHandler} name="AC" type="button">AC</button>
+        <button type="button" name="+/-" onClick={clickHandler}>+/-</button>
+        <button type="button" name="%" onClick={clickHandler}>%</button>
+        <button type="button" name="÷" className="operationsign" onClick={clickHandler}> ÷ </button>
+      </div>
+
+      <div className="symbole">
+        <button type="button" name="7" onClick={clickHandler}>7</button>
+        <button type="button" name="8" onClick={clickHandler}>8</button>
+        <button type="button" name="9" onClick={clickHandler}>9</button>
+        <button type="button" name="X" onClick={clickHandler} className="operationsign">X</button>
+      </div>
```

Comment:
- [ ] Kindly make sure the attribute `name` of this multiplication button is written in lowercase because now it is in uppercase and it causes errors which leads to website breaks and when we want to multiply we get errors. Please fix that.

   - ### ERROR
   
    ![image](https://user-images.githubusercontent.com/87197412/222412986-bc7ef018-aca5-44e5-a00a-1e29c00bfd87.png)

Reply:
@levy002 Now it is working perfectly

Reply:
Hello. That's good. If you are ready for a second review please go ahead and submit for re-review. If it works fine, the next reviewer will approve it right away